include/openthread.h

```diff
@@ -191,6 +191,19 @@ otInstance *otInstanceInit(void);
 #endif
 
 /**
+ * This function initializes OpenThread stack.
+ *
+ * This function initializes OpenThread stack with following steps:
+ *   1. Initialize settings subsystem;
+ *   2. Retrieve active operational dataset from non-volatile memory, and apply the network configuration;
+ *   3. Retrieve pending operational dataset from non-volatile memory.
+ *
+ * @param[in] aInstance A pointer to an OpenThread instance.
+ *
+ */
+void otInit(otInstance *aInstance);
```

Comment:
I have a question about where to put the `otPlatSettingsInit(otInstance *aInstance)`, since we need to call it before restoring the datasets, but it seems not proper to put it in the constructor of `MeshCoP::DatasetManager`, may I know your opinion? Thanks.